nx/exercises/exercises-1-20.livemd

```diff
@@ -0,0 +1,196 @@
+# Exercises: 1-20
+
+```elixir
+Mix.install([{:nx, github: "elixir-nx/nx", sparse: "nx"}])
+```
+
+## Introduction
+
+Inspired by the Python notebook _100 Numpy Exercises_:
+
+https://www.kaggle.com/code/utsav15/100-numpy-exercises/notebook
+
+## 1-10
+
+1. Install Nx in a Livebook (★☆☆)
+
+```elixir
+Mix.install([{:nx, github: "elixir-nx/nx", sparse: "nx"}])
```

Comment:
I think we should point to the latest hex version here.

Reply:
That was my first choice, but I got worried about it getting out of date.

Then again, maybe that's a good thing? If exercises are answered under version (say) 0.6, they might no longer be the best practice in later versions.

Reply:
It's better to use the Hex version. If it gets out of date, it's more explicit through the version

Reply:
[f9c005c](https://github.com/elixir-nx/nx/pull/1363/commits/f9c005cd5be16578574a3df51d0e2a348f379beb)